engine/src/main/scala/cromwell/engine/backend/runtimeattributes/CromwellRuntimeAttributes.scala

```diff
@@ -142,29 +152,67 @@ object CromwellRuntimeAttributes {
       case scala.util.Success(x) => x.to(MemoryUnit.GB).amount.successNel
       case scala.util.Failure(x) => x.getMessage.failureNel
     }
+    val walltime = validateWalltime(attributeMap.get(WALLTIME))
+    val queue = validateQueue(attributeMap.get(QUEUE))
     val bootDiskSize = validateBootDisk(attributeMap.get(BOOT_DISK))
-
-    (docker |@| zones |@| failOnStderr |@| continueOnReturnCode |@| cpu |@| preemptible |@| disks |@| memory |@| bootDiskSize) {
-      new CromwellRuntimeAttributes(attributes, _, _, _, _, _, _, _, _, _)
-    } match {
+    
+    /*
+     * scalaz.syntax.ApplicativeBuilder |@| (as of version 7.3, anyhow) runs out of steam at 12
+     * arguments so we can't just keep extending what Broad had here to add queue, walltime, etc.
+     * The chained <*> below basically does the same thing.
+     */
+    val curriedCtor = (CromwellRuntimeAttributes(_: Map[String, WdlValue], _: Option[String], _: Seq[String], _: Boolean, _: ContinueOnReturnCode, _: Int, _: Int, _: Seq[JesAttachedDisk], _: Double, _: String, _: Option[String], _: Int)).curried
```

Comment:
If you're willing to rename the `apply` on the companion object (which is more than ok with this reviewer), this could simplify to:

```scala
    val curriedCtor = (CromwellRuntimeAttributes.apply _).curried
```